### ISSUE TITLE
Style issue - popup windows [v.1.0.47]

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/NEOSDiscovery#195]
In Test: when clicking on a popup window from a record (cite this item, text me this item, email me this item, librarian view), there is a style issue causing text/fields to extend further than intended  

To replicate:

 - Click on any of these links in test

**Comment by pgwillia** [comment]
![screenshot from 2018-10-16 12-40-35](https://user-images.githubusercontent.com/1220762/47039286-d9fabb00-d140-11e8-89e5-5dbc02958159.png)
![screenshot from 2018-10-16 12-40-04](https://user-images.githubusercontent.com/1220762/47039287-d9fabb00-d140-11e8-9260-f5e35e04ac9f.png)

It looks like the header/footer with branding is creating the problem. Do you want these forms to go back to how they used to look?  Or keep the branding and fix the css?

**Comment by seanluyk** [comment]
@pgwillia I'd say it's more important to fix the css for these - it's most important for the branding in the header to persist

**Comment by nnunn** [comment]
The branding should not be coming through in those pop-up windows. I'm not sure why is happening. So they should look like they used to look - with no internal branding.

**Comment by nnunn** [comment]
Are the layouts somehow coming through into the pop-ups now? So strange.

**Comment by nnunn** [comment]
@pgwillia is this sufficient clarification? Let me know if there's anything I can do.

**Comment by pgwillia** [comment]
git bisect identified 
```
5a1a8fe64759a9058b87d4e9c63ff0044221369c is the first bad commit
commit 5a1a8fe64759a9058b87d4e9c63ff0044221369c
Author: dependabot[bot] <support@dependabot.com>
Date:   Wed Oct 10 15:29:09 2018 +0000

    Bump blacklight from 6.10.1 to 6.15.0

    Bumps [blacklight](https://github.com/projectblacklight/blacklight) from 6.10.1 to 6.15.0.
    - [Release notes](https://github.com/projectblacklight/blacklight/releases)
    - [Commits](https://github.com/projectblacklight/blacklight/compare/v6.10.1...v6.15.0)

    Signed-off-by: dependabot[bot] <support@dependabot.com>

:100644 100644 e1dda2708283793dc38fab09a4dd11a7e3a19490 d01edf0f0bbdf4e5d02819b3037594732699f396 M      Gemfile.lock
```

**Comment by nnunn** [comment]
I think I approved that one - didn't seem like it would cause issues.

**Comment by pgwillia** [comment]
@nnunn I'm still investigating.  This might turn out to be a red herring.  I'm not sure why the header/footer is on the modal forms either.

**Comment by pgwillia** [comment]
Yeah, it's definitely the blacklight gem bump.  Working on a PR to fix.

**Comment by nnunn** [comment]
great - thanks. Let me know what you find. I'd be interested in figuring out why that would happen.

**Comment by pgwillia** [comment]
My best guess is https://github.com/projectblacklight/blacklight/compare/v6.10.1...v6.15.0#diff-311f950985651c9e30301440331e2430 , https://github.com/projectblacklight/blacklight/compare/v6.10.1...v6.15.0#diff-0b79e35ce72565f0814a51fa527519aa or https://github.com/projectblacklight/blacklight/compare/v6.10.1...v6.15.0#diff-ae8c48d65bb99d96afd75a9d3ae58b42
But could be anywhere in here: https://github.com/projectblacklight/blacklight/compare/v6.10.1...v6.15.0

I'm just going to revert the blacklight bump until we can figure out how to address this.

**Comment by nnunn** [comment]
ok sounds good.
